l10n_it_fiscalcode/model/l10n_it_fiscalcode_mixin.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import models, fields
+
+
+class L10nItFiscalcodeMixin(models.AbstractModel):
+    _name = 'l10n_it_fiscalcode.mixin'
```

Comment:
https://github.com/OCA/l10n-italy/pull/2017#pullrequestreview-561539307

Reply:
Because this addon haven't `l10n_it_account` dependency.